addons/portal/static/src/scss/portal.scss

```diff
@@ -468,6 +468,7 @@ form label {
     #sidebar_content.card {
         border-left: 0;
         border-bottom: 0;
+        top: 10rem;
```

Comment:
Before this commit  when user scroll-down the page, whole sidebar content is scroll-up.
And after this commit portal sidebar content is scroll-up upto only 10rem, and it will fit on the page.

Reply:
@jem-odoo  There is one point in the task to sticky-top the sidebar_content so to align with a portal right side page on scroll we use this. Thanks